src/interface/src/app/services/auth.service.ts

```diff
@@ -173,6 +173,30 @@ export class AuthService {
         })
       );
   }
+
+  /** "Deletes" user from backend. The behavior of this command is to disable the user account,
+   *  not fully delete it, so data can be restored later if necessary.
+   */
+  deleteUser(user: User): Observable<boolean> {
+    return this.http
+      .post(
+        BackendConstants.END_POINT.concat('/users/delete/'),
+        {
+          email: user.email,
```

Comment:
side question: do we allow the user to make a new account with the same email or username if their previous account was deactivated?

Reply:
They wouldn't be able to, since the previous account still exists and can be reactivated by an admin.